pipeline_documentation.md

```diff
@@ -106,14 +105,14 @@ on installation and usage please see [here](README.md).
 
 ## Description of workflow steps
 
-> The workflow consists of four Snakemake files: A main `Snakefile` and an
-individual Snakemake file for each step in the workflow (the file preparation,
-the reads mapping, the miRNA quantification and the ASCII-style alignment
-pileups generation). The main `Snakefile` contains the configuration file
-validation along with the inclusion of the sub-workflows. Individual steps of
-the workflow are described briefly, and links to the respective software
-manuals are given. Parameters that can be modified by the user (via the samples
-table and the configuration file) are also described.
+> The workflow consists of five Snakemake files: A main `Snakefile` and an
+> individual Snakemake file for each step in the workflow (the genome resources
+> preparation, the reads mapping, the miRNA quantification and the ASCII-style
+> pileups generation). The main `Snakefile` contains the configuration file
+> validation along with the inclusion of the sub-workflows. Individual steps of
```

Comment:
Perhaps better: `along with the wiring of the "sub-workflows"` (better in quotes perhaps, because it's also a reserved word, no? Are they really subworkflows, as per Snakemake's definition? If so, then no quotes)

Reply:
According to the [Snakemake documentation](https://snakemake.readthedocs.io/en/stable/snakefiles/modularization.html#wrappers):

> In contrast to modularization strategies like include or subworkflows, the wrapper directive allows to re-wire the DAG of jobs. 

I guess I could use `module` instead if you like it better.

Reply:
If that's the correct terminology in Snakemake-speed, then yes :)

So something like that:

> The main `Snakefile` contains the configuration file validation and imports the various functional modules described below.